src/extra/gfx/gltf.cpp

```diff
@@ -13,12 +15,14 @@ namespace gfx {
 struct GLTFBlock {
   static inline Type PathInputType = CoreInfo::StringType;
   static inline Type ByteInputType = CoreInfo::BytesType;
-  static CBTypesInfo inputTypes() { return CoreInfo::AnyType; }
+  static inline Type TransformVarType = Type::VariableOf(CoreInfo::Float4x4Type);
+  static CBTypesInfo inputTypes() { return CoreInfo::AnyTableType; }
   static CBTypesInfo outputTypes() { return Types::DrawableHierarchy; }
 
   static CBParametersInfo parameters() {
     static Parameters params{
-        {"StaticPath", CBCCSTR("The static path to load a model from during warmup"), {CoreInfo::StringType}},
+        {"Transform", CBCCSTR("The transform variable to use"), {TransformVarType}},
```

Comment:
Is this to override/fix the default transform basically?

Reply:
The same way as we decided it would work with the normal Drawable.
Where if you pass (GLTF :Transform <variable>) it will capture the variable and move the drawable every time you change the variable.